lbrynet/lbrynet_daemon/LBRYDaemonServer.py

```diff
@@ -179,12 +361,35 @@ def render_GET(self, request):
     #     call.addErrback(lambda err: log.info("Error: " + str(err)))
     #     call.cancel()
 
```

Comment:
Good catch. It's in PEP 8, so I do follow this (the only thing I knowingly bend in PEP 8 in is line lengths)